index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
+        <nav class="menu-icon">
+            <h4 class="logo"><a href="#">My Logo</a></h4>
+            <img class="menubars" src="menu.png" alt="Menu"> 
+        </nav>
```

Comment:
- [ ] For semantic reasons and accessibility to your page, KIndly make sure your site `logo` and `navigation menu` are inside a `header` tag You can check this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Please make sure you grab these elements in a `header` and after the `main` can come next.